src/WrapperApp/components/NavDrawer/NavDrawer.test.ts

```diff
@@ -0,0 +1,40 @@
+import { Builder, By, WebDriver, until } from 'selenium-webdriver';
+import chrome from 'selenium-webdriver/chrome';
+
+describe('NavDrawer component', () => {
+  let driver: WebDriver;
+
+  beforeAll(async () => {
+    driver = await new Builder()
+      .forBrowser('chrome')
+      .setChromeOptions(new chrome.Options().headless())
+      .build();
+  }, 30000);
```

Comment:
What is the meaning of 30000 ? Is it 30000 miliseconds ?

Reply:
Can you add some comment lines explaining that ? Seeing raw numbers like 30000 make the code a bit unreadable.

Reply:
ok

Reply:
The question is what this line is in fact doing that it takes it more than 5 seconds to execute. I could imagine that `npm install` or `npm build` could take some time.
Good comment should contain the reasons behind the code.

Reply:
This line just creates an instance of chrome webdriver - I think it just takes some time because of low processing power provided in github actions, as default timeout was enough for the test to pass locally. Commands such as `npm install` and `npm run start` are launched seperately, before running tests

Reply:
OK, got it